src/main/flight/mixer.h

```diff
@@ -199,6 +200,8 @@ void filterServos(void);
 extern int16_t motor[MAX_SUPPORTED_MOTORS];
 extern int16_t motor_disarmed[MAX_SUPPORTED_MOTORS];
 
+bool motorLimitReached;
```

Comment:
`extern bool motorLimitReached` in .h file,
then define it (`bool motorLimitReached`) in .c file that owns it